fuzzers/064-gtp-channel-conf/Makefile

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
```

Comment:
The Makefile should handle the possibility to have different build directories for each part of the same family. This because the fuzzer must be run for each part, given the difference in frames for some tiles. See the Makefile for the [GTP_COMMON](https://github.com/SymbiFlow/prjxray/blob/c59c486959b48ae5e449816f077f9e249c7ac45a/fuzzers/063-gtp-common-conf/Makefile) which should similar

Reply:
Reworked to use `${BUILD_DIR}`